### PR TITLE
Machines view

### PR DESCRIPTION
**Pull request by jeffandersen**
Branched from https://github.com/manifoldco/torus-cli/pull/7

Will continue to update this as code review happens for machines create.

Sample output:

```
> torus machines view -h
NAME:
   torus machines view - Show the details of a machine

USAGE:
   torus machines view [command options] <identity>

OPTIONS:
   --org ORG, -o ORG  org to the machine will belongs to
```

```
> torus machines view 04buv5v36xecb5r390btvevkh0k10

ID:                  04buv5v36xecb5r390btvevkh0k10
Name:                api-test-machines-00cjftz6
Team(s):             machine, api-test-machines
State:               active
Created By:          jeff (Jeff Andersen)
Created On:          2016-10-30T12:56:21Z
Destroyed By:        -
Destroyed On:        -

TOKEN ID                             STATE         CREATED BY                  CREATED ON

04c5z24qauvxejreh3xkruyqd6zdc        active        jeff (Jeff Andersen)        2016-10-30T12:56:21Z

```

```
> torus machines view api-test-machines-00cjftz6

ID:                  04buv5v36xecb5r390btvevkh0k10
Name:                api-test-machines-00cjftz6
Team(s):             machine, api-test-machines
State:               active
Created By:          jeff (Jeff Andersen)
Created On:          2016-10-30T12:56:21Z
Destroyed By:        -
Destroyed On:        -

TOKEN ID                             STATE         CREATED BY                  CREATED ON

04c5z24qauvxejreh3xkruyqd6zdc        active        jeff (Jeff Andersen)        2016-10-30T12:56:21Z

```

### CODE REVIEW

**Comment by ianlivingstone** [comment]
The UI looks awesome, I'm excited!

We really should support viewing a machine by its name as well (we could do something fun like support providing the Machine ID, Machine Token ID, or Name via regex to look up a machine) as its much more likely for pets to be the primary use-case today (think CI you'd probably just call the machine Travis or something along those lines).

You can fetch a machine by name using `GET /machines?org_id=<org_id>&name=x` then perform the direct `GET /machines/:id` to get the tokens and such.

**Comment by jeffandersen** [comment]
@ianlivingstone added support to view by either name or ID

File: api/machines.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func createTokenSecret() (*base64.Value, error) {
 }
 
 // List machines in the given org and state
-func (m *MachinesClient) List(ctx context.Context, orgIDs []*identity.ID, state *string, teamID *identity.ID) ([]*apitypes.MachineSegment, error) {
+func (m *MachinesClient) List(ctx context.Context, orgIDs []*identity.ID, state *string, names *[]string, teamIDs *[]*identity.ID) ([]*apitypes.MachineSegment, error) {
```
Comment (ianlivingstone) [inline api/machines.go]: these endpoints only support a single instance today, so we should probably match the behaviour

**Comment by jeffandersen** [comment]
@ianlivingstone changes pushed

**Pull request merged**